### PR TITLE
Client 466

### PR DESCRIPTION
**Pull request by Comoque1**
### Describe the background of your pull request

- [x] Run tests as part of travis-ci build
- [x] Display coverage on coveralls
- [x] Added coverage badge on `readme.md`

### Additional context

Add any other context

### Governance 

- [x] Documentation is added
- [x] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

**Comment by Comoque1** [comment]
Note:
UI E2E tests are not configured to run as part of the build.

File: README.md [hunk 1]
```diff
@@ -6,10 +6,8 @@ It is in part inspired by the following article: https://blog.logrocket.com/buil
 
 The intention is that this repository contains over time a selection of React components for working with Cherrytwist instances, as well as a sample web client that uses those components.
 
-Build Status
 ![AKS dev CI/CD pipeline](https://github.com/cherrytwist/Client.Web/workflows/AKS%20dev%20CI/CD%20pipeline/badge.svg?branch=develop)
-
-Build Quality:
+[![Coverage Status](https://coveralls.io/repos/github/cherrytwist/Client.Web/badge.svg?branch=client-466)](https://coveralls.io/github/cherrytwist/Client.Web?branch=client-466)
```
Comment (valentinyanakiev) [inline README.md:10]: This (coverage status) needs to be taken from develop branch

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Seems OK, but needs to take the badges from develop for both build and coverage.

File: README.md (hunk 1 above)
Comment (Comoque1) [reply README.md]: Coverage status is now being taken from develop branch.

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**